aten/src/ATen/native/cuda/SpectralOps.cpp

```diff
@@ -234,6 +234,22 @@ void cufft_clear_plan_cache_impl(int64_t device_index) {
 namespace {
 constexpr int64_t cufft_max_ndim = 3;
 
+// "Large" here means a prime factor not special-cased by cuFFT
+bool has_large_prime_factor(int64_t n) {
+  constexpr int64_t first_large_prime = 11;
+  const std::array<int64_t, 4> prime_radices{{2, 3, 5, 7}};
```

Comment:
We compile with C++14 so I don't think the double braces are needed?

Reply:
I think all compilers accept single braces here, but technically it should be two since it's an aggregate initizalizer with an array member.

Reply:
OK